src/tests/ie_test_utils/common_test_utils/postgres_link.cpp

```diff
@@ -0,0 +1,935 @@
+// Copyright (C) 2022 Intel Corporation
```

Comment:
It may (and I think must) be used in all tools which are based on gtest. In such case we will significantly improve reporting of all gtest-based tests. Anyway, it easily turns on/off later, in runtime, per-executable. Did I answer on your question?